codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/EntityServiceSourceGenerator.scala

```diff
@@ -143,31 +143,35 @@ object EntityServiceSourceGenerator {
       className: String,
       interfaceClassName: String,
       entityType: String
+  ): Document = {
```

Comment:
(this *looks* like a lot of changes, but that's mainly because I split out the value entity and event sourced entity cases to their own methods)